app/redux-modules/user.js

```diff
@@ -114,9 +115,14 @@ export function setLoginStatus(status) {
 
 export function logout() {
   return async (dispatch) => {
-    await GoogleSignin.revokeAccess();
     dispatch({
-      type: LOGOUT
+      type: LOGOUT_ATTEMPT,
```

Comment:
What do you think about using a standard way for the actions? 
For example, as they do in the redux-offline library:
- LOGOUT_REQUEST
- LOGOUT_COMMIT
- LOGOUT_ROLLBACK (we are very optimistic and this never happens haha)